vendor/codefresh/DESCRIPTION

```diff
@@ -1 +1 @@
-Codefresh CLI provides a full and flexible interface to interact with Codefresh.
+Codefresh CLI
```

Comment:
Any reason to remove this?

Reply:
this is the result of `make -C vendor/codefresh update`